vulnerabilities-CVEd/CVE-2023-38035 - Ivanti Sentry - Auth Bypass.bcheck

```diff
@@ -0,0 +1,33 @@
+metadata:
+    language: v1-beta
+    name: "CVE-2023-38035 - Ivanti Sentry - Auth Bypass"
+    description: "Checks for CVE-2023-38035"
+    author: "Dolph Flynn"
+    tags: "CVE-2023-38035", "ivanti", "mobileiron", "OAST"
+
+
+define:
+    collatorator_payload = `{generate_collaborator_address()}/?{random_str(123)}`
+    trimmed_collaborator_payload = `{regex_replace ({collatorator_payload}, "(?<=^.{71}).*" , "")}` # trims payload to 71 characters!
+
+
+given host then
+    send request:
+        `POST /mics/services/MICSLogService HTTP/1.1
+Host: {base.request.url.host}
```

Comment:
Can you fix this indenting please?

Reply:
ignore me, I suspect it can't be resolved as we need the raw request.

Reply:
I had assumed this was the case and formatted accordingly, but from a brief test the logic seems to be that any newline is normalised to 0xd0xa (even within the HTTP body) and any leading space after a newline is stripped (even within the HTTP body). The latter seems somewhat overzealous to me.

With respect to this PR, I'm happy to update the formatting if required.